apps/frontend-manage/src/components/evaluation/BarChart.tsx

```diff
@@ -73,7 +87,21 @@ function BarChart({ data, showSolution }: BarChartProps): React.ReactElement {
               return rounded + 1
             },
           ]}
-          label={{ angle: -90, position: 'insideLeft', value: 'Antworten' }}
+          label={{
+            angle: -90,
+            position: 'insideLeft',
+            value: 'Antworten',
+            style: {
+              fontSize:
```

Comment:
Possible to reuse the object used above with a scaling factor? Keeps the relative distance consistent